scratch/patchExportModules.py

```
@@ -0,0 +1,34 @@
+                content = f.read()
+            
+            # Replace the usages
+            new_content = content.replace("formatNameInAppState", "formatNamePrintState")
+            
+            if new_content != content:
+                with open(file_path, 'w', encoding='utf-8') as f:
+                    f.write(new_content)
+                print(f"Patched {file_path}")
+            else:
+                print(f"No replacement occurred in {file_path}")
+        except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)